workspaces/preferences.md

```diff
@@ -40,9 +40,10 @@ need to access from your workspace.
 If necessary, you can regenerate your key. Be sure to provide your updated key
 to all of the services you use. Otherwise, they won't work.
 
-> The SSH public key uses the ECDSA key algorithm instead of the more commonly
-> used RSA key. This provides you with a smaller, more performant key with the
-> same level of security as RSA.
+> The SSH public keys use the [ED25519 key algorithm](https://datatracker.ietf.org/doc/html/rfc8709)
+> instead of the more commonly used RSA algorithm.
+> This provides you with a smaller, more performant key with the same level of
```

Comment:
I don't know the history of this statement but seems like we should be careful about making claims about "same level of security" unless we have a citation from a relevant authority. It might be better to use a phrase similar to Wikipedia's explanation of EdDSA - something like: "The SSH keys that Coder provisions use the [Ed25519 key algorithm], which was designed to be faster than predecessors such as RSA, with smaller key sizes, without sacrificing security"

@deansheather Do you have thoughts on this?

Reply:
GitLab says in their docs that ed25519 is better than RSA so we should just go with that and cite the same source https://docs.gitlab.com/ee/ssh/index.html#ed25519-ssh-keys

Reply:
Thanks, that's a great reference. The language in that document is definitely weaker than we had ("Available documentation suggests that ED25519 is more secure than RSA.") which I think is the right approach to take with such things

Reply:
Done, thanks for the reference @deansheather !